ruler/regular_shapes.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from typing import Tuple
+
+
+def rounded_square(
+    resolution: float,
+    phys_size: Tuple[float, float],
+    declared_mls: float,
+    angle: float = 0,
+    center: Tuple[float, float] = (0, 0)) -> np.ndarray:
+    '''
```

Comment:
Complete the docstring for this and the disc() function below?

Reply:
Sure!